Examples/UIExplorer/UIExplorerApp.android.js

```diff
@@ -38,6 +39,8 @@ const UIExplorerNavigationReducer = require('./UIExplorerNavigationReducer');
 const UIExplorerStateTitleMap = require('./UIExplorerStateTitleMap');
 const URIActionMap = require('./URIActionMap');
 
+UIManager.setLayoutAnimationEnabledExperimental(true);
```

Comment:
Can we just enable this for the specific UIExplorerApp that showcases layout animations?